pyslang/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2023-2024 Michael Popoloski
```

Comment:
Probably don't need this as a separate file, the parent directory has the same license.